core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.html

```diff
@@ -95,17 +99,20 @@
     color: #094142;
   }
 
-  .oppia-exploration-footer .uib-dropdown-menu.author-profile-dropdown-menu {
+  .oppia-exploration-footer .author-profile-dropdown-menu {
     border: none;
     border-bottom-left-radius: 0;
     border-bottom-right-radius: 0;
     box-shadow: 0 -6px 12px rgba(0, 0, 0, 0.176);
     float: left;
     list-style-type: none;
-    margin: 0 0 8px 0;
+    top: auto !important;
+    transform: none !important;
+    will-change: auto !important;
```

Comment:
Is there any way we can avoid `!important`?

Reply:
ngbDropdown directive adds inline styles to the ul element.
These styles were causing issues with the positioning.
Only way to override inline styles is using `!important`.

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
Ah I see. Thanks for explaining, can we add this in the comments so that future devs are aware? Thanks!

Reply:
Can you add it in a comment explaining this?

Reply:
done